integrations/color-mode.yml

```diff
@@ -1,11 +1,13 @@
 name: color-mode
 description: Dark and Light mode for NuxtJS with auto detection
-long_description: ''
+long_description: '🌑 Dark and 🌕 Light mode with auto detection made easy with NuxtJS'
```

Comment:
Shall we avoid emojis? It might make design issues when we use same DB for cli

Reply:
agree but if we take info from script it will auto add emojis, won't it?. Will manually remove any I see for now but just something to be aware of

Reply:
There is always a manual review process and it is not being overriden by sync (later-on only with nuxt meta)